app/javascript/containers/PortfolioBuildingContainer.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PortfolioBuildingInfoContainer from './PortfolioBuildingInfoContainer';
+import CategoryDetailsContainer from './CategoryDetailsContainer';
+import { get } from '../fetch/requester';
+import { connect } from "react-redux";
+import { questionDataPerCategory } from '../selectors/answersSelector';
+
+class PortfolioBuildingContainer extends React.Component {
```

Comment:
Can you add comments at the top of each container class explaining what it does and where it fits in to the portfolio view? That would help clear things up since a lot of these are named similarly